guests/templates/guests/band_agreement.html

```diff
@@ -116,7 +116,7 @@
     <div class="form-group">
       <label class="col-sm-3 control-label">Performer Count</label>
       <div class="col-sm-6">
-        <input class="form-control" type="number" name="performer_count" value="{{ guest_info.performer_count|default(guest.estimated_performer_count) }}" />
+        <input class="form-control" type="number" name="performer_count" min="1" value="{{ guest_info.performer_count|default(guest.estimated_performer_count) }}" />
```

Comment:
Our guest department found that a guest can technically ask for negative badges. This is silly, so I fixed it for bands too.